xpad.c

```diff
@@ -11,6 +11,7 @@
  *               2006 Adam Buchbinder <adam.buchbinder@gmail.com>
  *               2007 Jan Kratochvil <honza@jikos.cz>
  *               2010 Christoph Fritz <chf.fritz@googlemail.com>
+ *               2022 Daniel Nguyen <daniel.nguyen.1@ens.etsmtl.ca>
```

Comment:
this is no longer used. Tracking authors happens via git.

Reply:
I created my git account using my personal e-mail, but would like to use my student e-mail for this project since it is related to a project that I have with them. Is that going to be a problem?

Reply:
no, just add a signed-off line as described in the PR template link, where you put the email you want to use

Reply:
Fixed on commit 6dba352. 
Line removed and Signed-off-by added to PR details.